src/components/metrics/card/SensitivitySection/SensitivitySection.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component, PropTypes} from 'react'
+import {BarChart, ScatterPlot} from 'react-d3-components'
+import {sampleMean, sampleStdev, percentile, cutoff, sortDescending} from 'lib/dataAnalysis.js'
+import everpolate from 'everpolate'
+import './style.css'
+
+function importance(r2) {
+  if (r2 < 0.05) { return 'low' }
+  else if (r2 < 0.5) { return 'medium' }
+  else { return 'high' }
+}
+
+class Plot extends Component {
+  render() {
+    const {xSamples, ySamples, size, xLabel, yLabel} = this.props
+    const data = [{ customValues: _.zip(xSamples, ySamples) }];
+    const valuesAccessor = (s) => s.customValues
+    const xAccessor = (e) => e[0]
+    const yAccessor = (e) => e[1]
+
+    if (size === 'SMALL'){ return(
+          <ScatterPlot
```

Comment:
It feels like this would be cleaner if you had a function that went from size -> size dependent params and only one return statement. 

e.g. 

function displayParamsForSize(size) {
  if (size === 'SMALL') {
    return {width: 180, height: 68, margin: {top: 5, bottom: 9, left: 5, right: 3}, xTickArgument: 0, xTickSize: 1, ... }
  } else {
     ...
   }
}

... 
const {width, height, margin, ... } = displayParamsFrom(size)
return (<ScatterPlot data={data} width={width} ... />)
...